kahuna/public/stylesheets/main.css

```diff
@@ -1046,21 +1045,31 @@ FIXME: what to do with touch devices
     margin-right: 300px;
 }
 
-.image-info {
-    margin-bottom: 15px;
+.image-info__group {
+    padding: 10px;
+    border-bottom: 1px solid #565656;
+} 
+
+.image-info:last-child .image-info__group{
+    border-bottom: none;
+}   
+
+.image-info__group table {
```

Comment:
I'd generally agree although in the case of `border-collapse`, it only makes sense on a table. The `width: 100%` would be correct regardless of the tag on the other hand.